addons/hr_holidays/models/hr_leave.py

```diff
@@ -108,7 +108,7 @@ def default_get(self, fields_list):
 
     def _get_start_or_end_from_attendance(self, hour, date, employee):
         hour = float_to_time(float(hour))
-        holiday_tz = timezone(employee.tz or self.env.user.tz)
+        holiday_tz = timezone(employee.tz or self.env.user.tz or str(pytz.utc).lower())
```

Comment:
why not? 
```suggestion
        holiday_tz = timezone(employee.tz or self.env.user.tz or or 'utc')
```

Reply:
Hello @sofiagvaladze ,
Changes done. Please check it.